recipes/meg/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "meg" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lamyj/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
Any reason you don't use PyPI as source? `url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/meg-{{ version }}.tar.gz`

Reply:
I find it cleaner to consider PyPi and Conda as separate package repositories rather than have Conda repackage PyPi. If you prefer the latter, I'll change the URL to PyPi

Reply:
No, I don't think this is a requirement. PyPI is just the default for most Python packages.
The autotick-bot will automatically create a PR in conda-forge when a new release is published on PyPI. I don't know if this works when using a GitHub archive.